lib/lightning_web/live/user_live/common_components.ex

```diff
@@ -0,0 +1,113 @@
+defmodule LightningWeb.UserLive.CommonComponents do
+  use LightningWeb, :component
+
+  import PetalComponents.Table
+
+  attr :socket, :map, required: true
+  attr :users, :list, required: true
+  attr :live_action, :atom, required: true
+
+  attr :user_deletion_modal, :atom,
```

Comment:
`:atom` ? You maybe mean `:any` (or `:map`) ?

Reply:
It's the module name _()_